src/appBootstrapper.tsx

```diff
@@ -235,6 +237,26 @@ if (__DEBUG__ && module.hot) {
     module.hot.accept('./routes', () => render());
 }
 
+async function loadCustomJs() {
+    const customJsFiles = getServerConfig().custom_js_files;
+    if (!customJsFiles) {
+        return Promise.resolve();
+    }
+    getServerConfig().customJsFunctions = {};
```

Comment:
i don't think it makes sense to put the customJsFunctions on serverconfig.  They should be on the window object.

Reply:
Fixed in 96026be